skietbaan_fe/src/components/Groups.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react'
+
+ class Groups extends Component {
+     constructor(props){
+         super(props);
+         posts:[]
+     }
+     
+     componentWillMount() {
+        fetch('http://skietbaan.retrotest.co.za/api/User')
```

Comment:
Are you not using React Redux? Don't do http calls in `componentWillMount`. Use the React-Redux pattern and send the data through the redux state.